disaster.el

```diff
@@ -116,27 +124,26 @@
   :type 'string)
 
 (defcustom disaster-cflags (or (getenv "CFLAGS")
-                               "-march=native")
+                               (disaster--arch-flags))
   "Command line options to use when compiling C."
   :group 'disaster
   :type 'string)
 
 (defcustom disaster-cxxflags (or (getenv "CXXFLAGS")
-                                 "-march=native")
+                                 (disaster--arch-flags))
   "Command line options to use when compiling C++.!"
   :group 'disaster
   :type 'string)
 
 
 (defcustom disaster-fortranflags (or (getenv "FORTRANFLAGS")
-                                     "-march=native")
+                                     (disaster--arch-flags))
   "Command line options to use when compiling Fortran."
   :group 'disaster
   :type 'string)
 
 (defcustom disaster-objdump
-  (concat (if (eq system-type 'darwin) "gobjdump" "objdump")
-          " -d -M att -Sl --no-show-raw-insn")
+  "objdump -d -M att -Sl --no-show-raw-insn"
```

Comment:
On mac m1 we can't use gobjdump, we need the counterpart made for clang. I fixed that if it's a standard intel mac then gobjdump otherwise just objdump.